build.gradle

```diff
@@ -9,6 +9,7 @@ plugins {
 
 repositories {
     mavenCentral()
+    maven { url 'https://repository.jboss.org' }
 }
 
 sourceCompatibility = '1.8'
```

Comment:
should be 11 for being able to build.

Reply:
No as we want to keep Java8 as the lower lever as only 2020.3 seems to be compiled as Java11 as the target